plotly/tests/test_core/test_graph_objs/test_error_bars.py

```diff
@@ -41,6 +41,5 @@ def test_instantiate_error_y():
            width=5)
 
 
-@raises(PlotlyDictKeyError)
 def test_key_error():
-    ErrorX(value=0.1, typ='percent', color='red')
+    assert ErrorX(value=0.1, typ='percent', color='red') == {'color': 'red', 'typ': 'percent', 'value': 0.1}
```

Comment:
So, if we're deprecating `go.ErrorX` in favor of `go.scatter.ErrorX`, then we should probably have two tests here:
- One for the deprecated behaviour (errors are no longer thrown)
- Another for the new syntax, which should basically be the same test as before:
```
@raises(PlotlyDictKeyError)
def test_key_error():
    go.scatter.ErrorX(value=0.1, typ='percent', color='red')
```

And we should verify that the error that is raised remains `PlotlyDictKeyError`

Reply:
🚨 - Breaking Change